esrally/driver/runner.py

```diff
@@ -1103,6 +1107,101 @@ def __repr__(self, *args, **kwargs):
         return "delete-data-stream"
 
 
+class CreateComponentTemplate(Runner):
+    """
+    Execute the `PUT index template API <https://www.elastic.co/guide/en/elasticsearch/reference/current/indices-component-template.html>`_.
+    """
+
+    async def __call__(self, es, params):
+        templates = mandatory(params, "templates", self)
+        request_params = params.get("request-params", {})
+        for template, body in templates:
+            await es.cluster.put_component_template(name=template, body=body,
+                                                    params=request_params)
+        return len(templates), "ops"
+
+    def __repr__(self, *args, **kwargs):
+        return "create-component-template"
+
+
+class DeleteComponentTemplate(Runner):
+    """
+    Execute the `PUT index template API
+    <https://www.elastic.co/guide/en/elasticsearch/reference/current/indices-delete-component-template.html>`_.
+    """
+
+    async def __call__(self, es, params):
+        template_names = mandatory(params, "templates", self)
+        only_if_exists = params.get("only-if-exists", False)
+        request_params = params.get("request-params", {})
+
+        async def _exists(name):
+            return await es.transport.perform_request(
```

Comment:
What's the reason for this low-level API call? Is this (yet?) unsupported by the Python client? Can you please add a clarifying comment?

Reply:
Exactly its not currently supported. I'll import locally as well and add a comment.